Article/Article.js

```diff
@@ -3,18 +3,19 @@
 class Article {
   constructor(domElement) {
     // assign this.domElement to the passed in domElement
-    this.domElement;
+    this.domElement = domElement;
     // create a reference to the ".expandButton" class. 
-    this.expandButton;
+    this.expandButton = this.domElement.querySelector('.expandButton');
     // Using your expandButton reference, update the text on your expandButton to say "expand"
-    
+     this.expandButton.textContent = 'expand';
     // Set a click handler on the expandButton reference, calling the expandArticle method.
+    this.expandButton.addEventListener('click' , this.expandArticle);
     
   }
 
-  expandArticle() {
+  expandArticle(event) {
     // Using our reference to the domElement, toggle a class to expand or hide the article.
-
+    event.target.parentElement.classList.toggle('article-open');
```

Comment:
We talked about this in stand up, but the comment provides a clue to the correct way of implementing this method.